src/test/java/school/lemon/changerequest/java/io/pr1/task1/NumberSearchEngineImpTest.java

```diff
@@ -0,0 +1,64 @@
+package school.lemon.changerequest.java.io.pr1.task1;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+/**
+ * Created by User on 03.02.2017.
+ */
+public class NumberSearchEngineImpTest {
+    static NumberSearchEngineImp searchEngineImp;
+    static StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Not sure StringBuilder is needed. Just write it as constant. So JVM can handle `src/main/java` both on UNIX and windows systems, so there is no need to build path with separator here.